### PR TITLE
Show "average guesses" stat

### PR DESCRIPTION
**Pull request by iancanderson**
See https://twitter.com/katherinecodes/status/1490137584408100865

Screen Shot 2022-02-06 at 1 38 57 PM

### CODE REVIEW

File: src/components/EndGameModal.tsx [hunk 1]
```diff
@@ -10,12 +9,45 @@ type Props = {
...
+  avgGuessesPerGame: number
+}
+
+const avgGuessesPerGameGreatThreshold = 2.8
+const avgGuessesPerGameGoodThreshold = 3.0
+const avgGuessesPerGameOkayThreshold = 3.2
+
+const streakOkayThreshold = 25
+const streakGoodThreshold = 50
+const streakGreatThreshold = 100
```
Comment (iancanderson) [inline src/components/EndGameModal.tsx:27]: You'll probably want different thresholds 🤔

Comment (octokatherine) [inline src/components/EndGameModal.tsx:27]:
```suggestion
const streakOkayThreshold = 5
const streakGoodThreshold = 20
const streakGreatThreshold = 50
```
maybe this 🤔

**Review by octokatherine — APPROVED** [review]
love it, thanks!

File: src/App.tsx [hunk 2]
```diff
@@ -160,7 +164,6 @@ function App() {
   const isValidWord = (word: string): [boolean] | [boolean, string] => {
     if (word.length < 5) return [false, `please enter a 5 letter word`]
     if (difficultyLevel === difficulty.easy) return [true]
-    debugger
```
[the remark this answers is not in the document]
Comment (iancanderson) [reply src/App.tsx:163]: yeaaaa I added this "feature" in the `typescript` branch 😂  https://github.com/octokatherine/word-master/pull/99/files#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2R163

**Pull request merged**